common/catalog/common/haproxy.bom

```diff
@@ -110,7 +110,7 @@ brooklyn.catalog:
               ENTITY_ID: $brooklyn:attributeWhenReady("entity.id")
             command: |
               (
-              flock -o 9
+              flock 9
```

Comment:
Excellent SO discussion @aledsage, I overlooked it. 
There are interesting variations on the same theme there, but the conclusion appears the same: make sure that the file descriptor is not propagated to the child process.